packages/plugins/rubygem-foreman_remote_execution/rubygem-foreman_remote_execution.spec

```diff
@@ -1,10 +1,10 @@
 # template: foreman_plugin
 %global gem_name foreman_remote_execution
 %global plugin_name remote_execution
-%global foreman_min_version 3.9
+%global foreman_min_version 3.10
```

Comment:
you say you want to CP, but this will prevent CPing -- seems odd to bump min Foreman in a .z plugin release?